operator/v1/types_network.go

```diff
@@ -65,6 +65,13 @@ type NetworkSpec struct {
 	// +optional
 	DeployKubeProxy *bool `json:"deployKubeProxy,omitempty"`
 
+	// disableNetworkDiagnostics specifies whether or not periodic
+	// network connectivity checks should be disabled or not. If unset, this
+	// property defaults to 'false' and network diagnostics is enabled.
+	// +optional
+	// +kubebuilder:default:=false
+	DisableNetworkDiagnostics bool `json:"disableNetworkDiagnostics,omitempty"`
```

Comment:
omitempty is not needed as false is the default anyway.

Am curious, when is this disabled?

Reply:
Also a reference of what "network diagnostics" is and where to find the result? Some CR ?

Reply:
We won't set it disabled ourselves, we intend to run diagnostics by default.
But then customers may not want to have the additional workload of running network diagnostics, thus it's up to them.

Reply:
Would mention that in the description. "to reduce additional load".

Reply:
done